src/test/identity-manager/WorldIDIdentityManagerTest.sol

```diff
@@ -54,9 +55,9 @@ contract WorldIDIdentityManagerTest is WorldIDTest {
     uint256 internal constant SNARK_SCALAR_FIELD =
         21888242871839275222246405745257275088548364400416034343698204186575808495617;
 
-    // StateBridgeProxy mock
-    SimpleStateBridge internal stateBridge;
-    address internal stateBridgeProxy;
+    // StateBridgemock
+    SimpleStateBridge internal simpleStateBridge;
+    IBridge internal stateBridge;
```

Comment:
Do you ever need it to be typed as a `SimpleStateBridge`? As best I can tell we only ever use it as `IBridge`, so this two-layer construction is unnecessary.

Reply:
well I just need a mock which I can later call through an interface